src/blocks/shared/variables.scss

```diff
@@ -1,10 +1,43 @@
-$html-font-size: 16px;
-
-// Colors
-$black: #000;
-// gray
-$gray-300: #edeeef;
-$gray-600: #586069;
-$gray-700: #444d56;
-// blue
-$blue-550: #2596db;
```

Comment:
I like SCSS variables for the same reason and more so because you don't have to use `var()` everywhere. However, I've seen generally recommended to use CSS variables where possible, and most of the code was in CSS variables already.

We can look into that if it becomes important.